.travis.yml

```diff
@@ -1,8 +1,24 @@
 language: node_js
 node_js:
   - "8"
-
 script:
   - yarn run lint
-
+  - yarn run generate-example-images
+  - yarn run build
 cache: yarn
+deploy:
```

Comment:
Do we need to say that we only want to do this on master and only for tagged commits?

Reply:
Tagged commits makes more sense. Updating it.

Reply:
see below